api/api/V1/Station.ts

```diff
@@ -152,6 +153,41 @@ export default function (app: Application, baseUrl: string) {
     }
   );
 
+  /**
+   * @api {get} /api/v1/stations/:id/recordings-count Get count of recordings for a station by id
+   * @apiName GetRecordingsCountForStation
+   * @apiGroup Station
+   * @apiDescription Get the count of recordings associated with a station by its id
+   *
+   * @apiUse V1UserAuthorizationHeader
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiInterface {apiSuccess::ApiRecordingsCountResponseSuccess} count Number of recordings associated with the station
+   * @apiUse V1ResponseError
+   */
+  app.get(
+    `${apiUrl}/:id/recordings-count`,
+    extractJwtAuthorizedUser,
+    validateFields([
+      idOf(param("id")),
+      query("view-mode").optional().equals("user"),
+    ]),
+    async (request: Request, response: Response) => {
+      const stationdId = request.params.id;
```

Comment:
Strictly speaking should probably be making sure the user has access to the station – not that we're going to leak much data here, but still...